### PR TITLE
[BUGFIX] LLD - Receive/send deeplinks edge cases

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### 📝 Description

Fix Receive/send deeplinks edge cases 

### ❓ Context

- **Impacted projects**: `LLD` 
- **Linked resource(s)**: [LIVE-5588] [LIVE-7843]

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-5588]: https://ledgerhq.atlassian.net/browse/LIVE-5588?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIVE-7843]: https://ledgerhq.atlassian.net/browse/LIVE-7843?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts [hunk 1]
```diff
@@ -198,10 +198,14 @@ export function useDeepLinkHandler() {
           const modal =
             url === "send" ? "MODAL_SEND" : url === "receive" ? "MODAL_RECEIVE" : "MODAL_DELEGATE";
           const { currency, recipient, amount } = query;
-          if (!currency || typeof currency !== "string") return;
+
           if (url === "delegate" && currency !== "tezos") return;
-          const c = findCryptoCurrencyByKeyword(currency.toUpperCase()) as Currency;
-          if (!c || c.type === "FiatCurrency") {
+
+          const c = findCryptoCurrencyByKeyword(
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts:204]: I know it was named like this before but could you rename it to something more readable, please? Like `c` => `foundCurrency`

**Review by KVNLS — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts (hunk 1 above)
Comment (mcayuelas-ledger) [reply apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts:204]: yes :)

**Review by cgrellard-ledger — APPROVED** [review]
